### PR TITLE
docs: update DM master slave switch procedure

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 
Update docs about dm master-slave switch procedure

### What is changed and how it works?
no code

### CODE REVIEW

File: docs/dm/zh_CN/maintenance/master-slave-switch.md [hunk 1]
```diff
@@ -15,6 +15,8 @@
 
 1. 使用 `query-status` 确认 relay 已经追上切换前的 master（`relayCatchUpMaster`）
 2. `stop-task` 停止运行中的任务
-3. 变更 DM-worker 配置，使用 ansible 滚动升级 DM-worker
-4. 更新 `task.yaml` 配置，更新 `mysql-instances / config` 配置
-5. `start-task` 重新启动任务
+3. 上游 master-slave 执行切换
+4. 如果 DM-master 配置的 mysql-instance 为 `ip:port` 形式，更新 `dm-master.toml` 的 `deploy / mysql-instance` 配置，然后使用 `update-master-config` 更新 DM-master
```
Comment (csuzhangxc) [inline docs/dm/zh_CN/maintenance/master-slave-switch.md:19]: why need to update `mysql-instance`? even it is the format of `ip:port`, it is still an **identifier**. 
if we update the `mysql-instance`, the checkpoints will also need to be updated.

when we updating task.yaml, we only need to update the MySQL config, not `instance-id`

Comment (amyangfei) [reply docs/dm/zh_CN/maintenance/master-slave-switch.md:19]: In MySQL master-slave or master-master architecture, binlog position is not updated in slave/standby instance. So if we reuse the old `instance-id` and checkpoints in dm_meta, the binlog position is not right.

Comment (amyangfei) [reply docs/dm/zh_CN/maintenance/master-slave-switch.md:19]: probably we need to refine this procedure to guarantee `relay binlog postion` and `checkpoints in dm_meta` both have the right value based on `upstream MySQL binlog position`

Comment (csuzhangxc) [reply docs/dm/zh_CN/maintenance/master-slave-switch.md:19]: we have added a relay sub dir index part in checkpoint's binlog name.
eg. `000002` in `mysql-bin|000002.000003`

Comment (amyangfei) [reply docs/dm/zh_CN/maintenance/master-slave-switch.md:19]: got it

**Pull request closed without merging**